src/displayBackend/wayland/FrameBuffer.hpp

```diff
@@ -158,6 +159,21 @@ class DrmBuffer : public WlBuffer
 			  uint32_t width, uint32_t height, uint32_t pixelFormat);
 };
 
+/***************************************************************************//**
+ * Linux dmabuf buffer class.
+ * @ingroup wayland
+ ******************************************************************************/
+class LinuxDmabufBuffer : public WlBuffer
```

Comment:
DmabufBuffer -> DmaBuffer?

Reply:
linux_dmabuf is in the name of the protocol, to be consistent I use LinuxDmabufBuffer